beater/client.go

```diff
@@ -29,7 +29,7 @@ func isServerUp(secure bool, host string, numRetries int, retryInterval time.Dur
 
 	for i := 0; i <= numRetries; i++ {
 		if check() {
-			logp.Info("HTTP Server ready")
+			logp.NewLogger("HTTP Client").Info("HTTP Server ready")
```

Comment:
While there is absolutely no requirement to use single word keys, conventionally we have named the loggers using lowercase and no spaces like `http_client` or `httpclient` or `http.client`.